tests/test_configupdater.py

```diff
@@ -1333,3 +1334,129 @@ def test_comments_in_multiline_values():
     parser.read_string(dedent(error_case_if_no_value))
     updater.read_string(dedent(error_case_if_no_value))
     assert str(updater) == dedent(error_case_if_no_value)
+
+
+def test_modify_multiline_value():
+    ml_value = """\
+    [metadata]
+    classifiers =
+        Development Status :: 3 - Alpha
+        #Development Status :: 4 - Beta
+        #Development Status :: 5 - Production/Stable
+        Programming Language :: Python :: 3 :: Only
+        Programming Language :: Python :: 3
+        Programming Language :: Python :: Implementation :: CPython
+        Programming Language :: Python :: Implementation :: PyPy
+        License :: OSI Approved :: MIT License
+    """
+    ml_value = dedent(ml_value)
+    updater = ConfigUpdater()
+    updater.read_string(ml_value)
+
+    with pytest.raises(ModifyMultilineValueError):
+        updater["metadata"]["classifiers"].value = "new_value"
```

Comment:
Should we check the existing value for multi-line or check the replacement?

In this case I would argue that `.value = "new_value"` is harmless, but `.value = "new\nvalue"` is problematic.

Reply:
Good point! I'll change it accordingly.

Reply:
On second thought. While `updater["metadata"]["classifiers"].value = "new_value"` is fine how can I make sure that `updater["metadata"]["classifiers"].value += "new_value"` fails with `ModifyMultilineValueError`?

Reply:
I think it will, won't it?

Let assume `value = "multi\nline\string"`, if I do `value += "new_value"`, it should be the same as `value = value + "new_value"` right?

In that case `value + "new_value" = "multi\nline\stringnew_value"` which should error, isn't it?

Reply:
Doh,... yeah. Sorry, that was stupid. I am still thinking in references.

Reply:
I am not sure, but the tests will definitely tell!

> I am still thinking in references.

Have you been playing with C++ lately? 🤣